wiki/Tournaments/GBC/GBC_2022_Autumn/en.md

```diff
@@ -190,16 +194,19 @@ The following rules are changed to consistent with rules on `MWC 4k 2022` :
 8. At all times, the captains may communicate with each other via multiplayer chatrooms or voice chat. 
    - Other players will be muted during the Drafting phase.
 9. After the Drafting phase, captains may choose, with the approval of the host, to delegate their role to other prospective team members.
-10. Bad behaviour during the Drafting phase will be seen as disrupting the match by foul play.
+10. Captains may also choose to rename their team name after the Drafting phase.
+    - The default name of team is: `Team {X}`, `{X}` is the abbreviation of captain's name.
+    - Team names can't be longer than **12** characters (Chinese character is considered as 1).
```

Comment:
yes, usually in the program a Chinese character would be considered as 2. since the team's name will using on room settings, I think a clear Chinese character still considered as 1 is necessary.
so I will only modify the `cannot` part. if you agree with that.

Reply:
ok i've edited the suggestion

Reply:
got it